src/debuginfo.cpp

```diff
@@ -774,9 +775,11 @@ static int lookup_pointer(DIContext *context, jl_frame_t **frames,
 
     int fromC = (*frames)[0].fromC;
     int n_frames = inlineInfo.getNumberOfFrames();
-    if (n_frames == 0)
+    if (n_frames == 0) {
+        JL_UNLOCK(&codegen_lock);
```

Comment:
Shouldn't it be fine to unconditionally use the `_NOGC` version of the macros here?